src/components/frame/SubscribedColoniesList/SubscribedColoniesList.tsx

```diff
@@ -20,17 +20,9 @@ const MSG = defineMessages({
 });
 
 const SubscribedColoniesList = () => {
-  const { user, userLoading, updateUser } = useAppContext();
-  const canInteractWithNetwork = useCanInteractWithNetwork();
+  const { user, userLoading } = useAppContext();
   const isMobile = useMobile();
 
-  /* Ensures colony list is up-to-date post create colony flow. */
-  useEffect(() => {
-    if (updateUser) {
-      updateUser(user?.walletAddress);
-    }
-  }, [user, updateUser]);
-
```

Comment:
Removed as of the following comment as it is now redundant - https://github.com/JoinColony/colonyCDapp/pull/83/files#r1029539075